app/frontend/src/features/search/SearchResult.tsx

```diff
@@ -16,77 +18,103 @@ import {
   },
 }));
 
-export default function SearchResult({ user }: { user: User.AsObject }) {
+interface SearchResultProps {
+  className?: string;
+  id?: string;
+  user: User.AsObject;
+  onClick: (user: User.AsObject) => void;
+  highlight?: boolean;
+}
+
+export default function SearchResult({
+  className,
+  id,
+  user,
+  onClick,
+  highlight = false,
+}: SearchResultProps) {
   const classes = useStyles();
   return (
-    <Link to={routeToUser(user.username)} className={classes.root}>
-      <Card className={classes.card}>
-        <CardActionArea>
-          <CardContent>
-            <UserSummary user={user}>
-              <div className={classes.statusLabelWrapper}>
-                <div>
-                  <CouchIcon />
-                  <Typography
-                    className={classes.statusLabel}
-                    display="inline"
-                    variant="subtitle1"
-                    color="primary"
-                  >
-                    {hostingStatusLabels[user.hostingStatus]}
-                  </Typography>
-                </div>
-                <div>
-                  <LocationIcon />
-                  <Typography
-                    className={classes.statusLabel}
-                    display="inline"
-                    variant="subtitle1"
-                  >
-                    {meetupStatusLabels[user.meetupStatus]}
-                  </Typography>
-                </div>
+    <Card
+      id={id}
+      className={classNames(classes.card, classes.root, className)}
+      onClick={() => onClick(user)}
+      elevation={highlight ? 4 : undefined}
+    >
+      <CardActionArea>
+        <CardContent>
+          <UserSummary user={user} avatarIsLink={false} compact>
+            <div className={classes.statusLabelWrapper}>
+              <div>
+                <Hidden smDown>
+                  <CouchIcon className={classes.statusIcon} />
+                </Hidden>
+                <Typography
+                  className={classes.statusLabel}
+                  display="inline"
+                  variant="body1"
+                  color="primary"
+                >
+                  {hostingStatusLabels[user.hostingStatus]}
+                </Typography>
+              </div>
+              <div>
+                <Hidden smDown>
+                  <LocationIcon className={classes.statusIcon} />
+                </Hidden>
+                <Typography
+                  className={classes.statusLabel}
+                  display="inline"
+                  variant="body1"
+                >
+                  {meetupStatusLabels[user.meetupStatus]}
+                </Typography>
               </div>
-            </UserSummary>
+            </div>
+          </UserSummary>
+          <Hidden mdUp>
+            <LinesEllipsis
+              text={aboutText(user)}
+              maxLine={4}
+              component="p"
+              className={classes.about}
+            />
+          </Hidden>
```

Comment:
This seems to display in a different font to what we got in the theme when on breakpoints below md. Probably need to use `Typography` for the `component` prop?

Reply:
It only accepts html tags. But I'll put the typography css in `classes.about`